File: 3. Sophomore_HKI/OOP/OOP_CUOIKI/QuanLyShopQuanAo (C#)/LoaiKhachHang.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyShopQuanAo
{
    public class LoaiKhachHang
    {
        public string maLoaiKhachHang { get; set; }     // primary key
        public string tenLoaiKhachHang { get; set; }

        // constructor
        public LoaiKhachHang() { }
        public LoaiKhachHang(LoaiKhachHang loaiKhachHang)
        {
            this.maLoaiKhachHang = loaiKhachHang.maLoaiKhachHang;
            this.tenLoaiKhachHang = loaiKhachHang.tenLoaiKhachHang;           
        }
        public LoaiKhachHang(string maLoaiKhachHang, string tenLoaiKhachHang)
        {
            this.maLoaiKhachHang = maLoaiKhachHang;
            this.tenLoaiKhachHang = tenLoaiKhachHang;
        }
    }
}

File: 3. Sophomore_HKI/OOP/OOP_CUOIKI/QuanLyShopQuanAo (Java)/src/mypack/LoaiKhachHang.java
package mypack;

public class LoaiKhachHang {

	public String maLoaiKhachHang; // primary key
	public String tenLoaiKhachHang;
	
	// constructor
	public LoaiKhachHang() { }
    public LoaiKhachHang(LoaiKhachHang loaiKhachHang)
    {
        this.maLoaiKhachHang = loaiKhachHang.maLoaiKhachHang;
        this.tenLoaiKhachHang = loaiKhachHang.tenLoaiKhachHang;           
    }
    public LoaiKhachHang(String maLoaiKhachHang, String tenLoaiKhachHang)
    {
        this.maLoaiKhachHang = maLoaiKhachHang;
        this.tenLoaiKhachHang = tenLoaiKhachHang;
    }
    
    // getter - setter
	public String getMaLoaiKhachHang() {
		return maLoaiKhachHang;
	}
	public void setMaLoaiKhachHang(String maLoaiKhachHang) {
		this.maLoaiKhachHang = maLoaiKhachHang;
	}
	public String getTenLoaiKhachHang() {
		return tenLoaiKhachHang;
	}
	public void setTenLoaiKhachHang(String tenLoaiKhachHang) {
		this.tenLoaiKhachHang = tenLoaiKhachHang;
	}
}
